### PR TITLE
Added date to dayCard

### PR DESCRIPTION
**Pull request by fullfatwasabi**
1. Changed order of dayCards such that the next 7 days (including the current one) would be displayed.
1. Added date to dayCards.

Screenshot taken on Tuesday:
Capture

After "change da/2021-12-25":
Capture
Adding shift to 31/12 does add to the Friday panel in this picture.

### CODE REVIEW

File: src/main/java/seedu/address/ui/DayCard.java [hunk 1]
```diff
@@ -29,20 +33,25 @@
...
     /**
      * Creates a {@code PersonCode} with the given {@code Person} and index to display.
      */
-    public DayCard(DayOfWeek day, ObservableList<Person> stafflist, Period currentPeriod) {
+    public DayCard(int dayCardNumber, ObservableList<Person> stafflist, Period currentPeriod) {
         super(FXML);
-        this.day = day;
-        dayLabel.setText(day.toString().substring(0, 3));
+        this.dayCardNumber = dayCardNumber;
+        DayOfWeek day = LocalDate.now().getDayOfWeek().plus(dayCardNumber);
```
Comment (Gabau) [inline src/main/java/seedu/address/ui/DayCard.java:49]: Should use the period input to change as the we have a command to change which week to look at.
I think the period::toList() function would help. 
Maybe retain the DayOfWeek input to compare with period::toList?

**Review by Gabau — CHANGES_REQUESTED** [review]
LGTM, but i do not think that it takes the change command into consideration, should just keep the DayOfWeek input and find the date to display?

File: src/main/java/seedu/address/ui/DayCard.java (hunk 1 above)
Comment (fullfatwasabi) [reply src/main/java/seedu/address/ui/DayCard.java]: Aighto, forgot about the Change command, have just updated it, if you wanna take a look.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 2]
```diff
@@ -510,17 +507,4 @@ public static Period extractPeriodDates(ArgumentMultimap argMultimap) throws Par
         return new LocalDate[]{currentDate.minusDays(diffFromMonday),
                 currentDate.plusDays(diffFromSunday)};
     }
-
-    public static Period getWeekPeriodFromDate(LocalDate date) {
-        LocalDate[] init = getDateArrayOfTheWeek(date);
```
Comment (Gabau) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:515]: Can't you use this method to get the one week?

File: src/main/java/seedu/address/model/person/Period.java [hunk 3]
```diff
@@ -283,6 +270,24 @@ public static boolean isValidPeriodString(String toTest) {
...
+    public LocalDate getStartDate() {
+        return startDate;
+    }
+
+    /**
+     * Returns a Period that represents a week that starts from the LocalDate provided.
+     *
+     * @param firstDate first date of the week.
+     */
+    public static Period oneWeekFrom(LocalDate firstDate) {
```
Comment (Gabau) [inline src/main/java/seedu/address/model/person/Period.java:287]: This does not obtain from monday to sunday exact as specified? Why not use the ParserUtil::getWeekPeriodFromDate?

**Review by Gabau — CHANGES_REQUESTED** [review]
I don't understand why the method to get the period was changed? Could have just used to Period::toList() method? Do you think it is better to have the duration be from different dates instead of being fixed from monday to sunday?

File: src/main/java/seedu/address/model/person/Period.java (hunk 3 above)
Comment (fullfatwasabi) [reply src/main/java/seedu/address/model/person/Period.java:287]: I changed it from [Monday to Sunday] to [Today to next week] It doesn't make sense to display the next Monday behind today if its Tuesday now isn't it??

Comment (Gabau) [reply src/main/java/seedu/address/model/person/Period.java:287]: As in currently, all the shift commands are doing it for the current week, as in monday to sunday. If we change it to display the next seven days including current, shouldn't we change the others as well?

Comment (Gabau) [reply src/main/java/seedu/address/model/person/Period.java:287]: Should probably modify ParserUtil::getDateArrayOfWeek instead

**Review by mweeruien — APPROVED** [review]
LGTM! :>

File: docs/UserGuide.md [hunk 4]
```diff
@@ -384,12 +384,13 @@ Format: `exit`
 
 #### Viewing schedule for the week: `change`
 
-The schedule for the next 7 days are shown under the schedule tab by default. The week
-can be changed using the `change` command.
+the change command enables the user to view a different week than the default one, which is the next seven days.
```
Comment (Gabau) [inline docs/UserGuide.md:387]: Missing capital letter for start of sentence.
```suggestion
The change command enables the user to view a different week than the default one, which is the next seven days.
```

**Review by Gabau — CHANGES_REQUESTED** [review]
LGTM, one problem with the formatting of the user guide.

**Review by Gabau — APPROVED** [review]
LGTM

**Pull request merged**